napari/_qt/_tests/test_app.py

```diff
@@ -33,6 +33,10 @@ def get_app_id():
 
 def test_run_outside_ipython(qapp, monkeypatch):
     """Test that we don't incorrectly give ipython the event loop."""
+    from IPython.core.interactiveshell import InteractiveShell
+
+    InteractiveShell._instance = None
```

Comment:
```suggestion
    assert hasattr(InteractiveShell, "_instance ")
    InteractiveShell._instance = None
```

to detect removes of `_instance` attribute?

Reply:
no need, the metaclass ensure `_instance` always exists. 

```
>>> from IPython.core.interactiveshell import InteractiveShell
>>> del InteractiveShell._instance
>>> InteractiveShell._instance is None
True
```